CHANGELOG.md

```diff
@@ -39,6 +39,8 @@ support had been officially deprecated in `v38`.  Please use functional componen
 
 * Improvements to exception serialization to better handle `LocalDate` and similar custom JS
   classes.
+* BlueprintJS' [`EditableText`](https://blueprintjs.com/docs/#core/components/editable-text) component brought into framework.  Can now be imported with:
+  `import {editableText, EditableText} from '@xh/hoist/kit/blueprint';`
```

Comment:
I'd suggest the title of this PR as a succinct one-liner for the CL - we haven't been overly emphasizing the use of these components directly out of the kit, and we haven't done any additional integration work. 

Hopefully developers can figure out the import syntax 😄 

I'll just go ahead and make that tweak in the interest of merging w/o you needing to go through another cycle.  Happy to discuss though, as always.